config/samples/config_v1alpha1_store_oras_k8secretAuth.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   name: oras
   parameters:
-    cosign-enabled: false  
+    cosign-enabled: true  
```

Comment:
```suggestion
    cosignEnabled: true  
```

is this used?

Reply:
Thanks for catching this. The `cosignEnabled` field isn't used for this specific test associated with this. I will just remove the field from this entirely.